### ISSUE TITLE
Pierre can die prematurely

### ISSUE THREAD

**Issue by vvs-** [opentomb/OpenTomb#451]
Pierre can be killed by e.g. slamming doors early in _Tomb of Tihocan_.

**Comment by TeslaRus** [comment]
not every Pierre's are equally lucky

**Comment by vvs-** [comment]
Right. I've seen that there is a Lua code that should prevent this by comparing the room to 110, but for some reason it doesn't work. Also, Pierre leaves too many loot for Lara, it's a cheating 😛

**Comment by TeslaRus** [comment]
in level2 wolves can die by darts =)

**Comment by vvs-** [comment]
Yeah. And in some levels enemies can die from water or lava too.

**Comment by TeslaRus** [comment]
Water kill in GTA 3 and Vice City too ;-) Lava kill everyone

**Comment by TokyoSU** [comment]
not only pierre but all entities can be die:
bat can be killed by an other bat if he attack and the other bat are in front.
same as crocodile and wolf. 
- need to dmg only lara, probably check collision before dmg i'm.
